express-notes-api/index.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable no-console */
+const express = require('express');
+const data = require('./data');
+const fs = require('fs');
+const app = express();
+const jsonMiddleware = express.json();
+const notes = data.notes;
+const notesArr = [];
+
+app.get('/api/notes', (req, res) => {
+  res.json(notesArr);
+});
```

Comment:
If you start the server and send it a `GET /api/notes`, you get back an empty array, which is not accurate.